Makefile

```diff
@@ -375,7 +362,8 @@ clean-pyc:
 .PHONY: clean-gui
 clean-gui:
 	- rm -rf jujugui/static/gui/build
-	- rm -rf fast-babel
```

Comment:
So I gather things are just faster now than they used to be? Otherwise I'm concerned about removing this.

Reply:
This being the fast-babel stuff, not this particular `rm` statement.

Reply:
`fast-babel` was always a hack to get around a limitation in Make being that it won't tell you the list of changed files, only one at a time. Now that I've moved that logic into the transpile.js it is always running the multi-threaded transpile process.